app/backend/templates/reference_reminder.md

```diff
@@ -0,0 +1,17 @@
+---
+subject: "You have {{ time_left_text|couchers_escape }} to write a reference for {{ other_user.name|couchers_escape }}!"
+---
+
+{% from "macros.html" import button, link, support_email %}
+
+Hi {{ user.name|couchers_escape }}!
+
+You have {{ time_left_text|couchers_escape }} left to write a reference for {{ other_user.name|couchers_escape }} from when you {% if surfed %} surfed with {% else %} hosted {% endif %} them.
+
+To write a reference, go to the Messaging tab and locate the request, then click on the "Write Reference" button at the bottom of the conversation.
```

Comment:
Link to messaging (or the correct message even better if possible)?

Also, should available references eventually get its own page on the website?

Reply:
> Link to messaging (or the correct message even better if possible)?

It's possible but a bit annoying. I'm hoping soon we'll have pending references to write on the dashboard, so we can just link there.

Reply:
I think a link to just messaging for now then. It's kind of annoying "Reminder to do this thing" and not even provide a general link to do it